config/product/logging.js

```diff
@@ -36,4 +40,9 @@ export function init(store) {
     route:      { name: 'c-cluster-logging' },
     exact:       true,
   });
+
+  headers(LOGGING.FLOW, [STATE, NAME_COL, OUTPUT, CLUSTER_OUTPUT, CONFIGURED_PROVIDERS, AGE]);
+  headers(LOGGING.OUTPUT, [STATE, NAME_COL, LOGGING_OUTPUT_PROVIDERS, AGE]);
+  headers(LOGGING.CLUSTER_FLOW, [STATE, NAME_COL, CLUSTER_OUTPUT, CONFIGURED_PROVIDERS, AGE]);
+  headers(LOGGING.CLUSTER_OUTPUT, [STATE, NAME_COL, LOGGING_OUTPUT_PROVIDERS, AGE]);
```

Comment:
Namespaced types (which, for some reason, is still all of them including the "cluster" ones) need a NAMESPACE column after name.